src/Node/FS.purs

```diff
@@ -7,6 +17,32 @@ import Prelude
 --
 foreign import data FS :: !
 
+foreign import data FileDescriptor :: *
+
+data FileFlags = R | R_PLUS | RS | RS_PLUS
+               | W | WX | W_PLUS | WX_PLUS
+               | A | AX | A_PLUS | AX_PLUS
+
+instance showFileFlags :: Show FileFlags where
```

Comment:
I think this should just be a normal monomorphic function, not a Show instance. If FileFlags is to have a Show instance then it should be show R = "R"; show R_PLUS = "R_PLUS"; ... and so on, so that the Show instance produces code that can be evaluated to give back the original value.